Python/courses.md

```diff
@@ -7,4 +7,5 @@
 - [Python with Django](https://goo.gl/DVM0m5) - Django over the python.
 - [Programming fundamental with python](https://goo.gl/JEdLMw) - Intermediate level course on python data structure and basic topics.
 - [Developing Scalable app with Python](https://goo.gl/EwWZDW) - How to use Google app engine along with python to create scalable webApp. 
-
+-[Python Ultimate Course- Udemy](https://goo.gl/GRuvrL) - Python course by Udemy to build concepts.
+-[Networking in Python](http://www.tutorialspoint.com/python/python_networking.htm) - A good tutorial to implement networking in python.
```

Comment:
As discussed with @PSanni, he suggested me to add the Python Udemy course under course section. Networking in Python is a tutorial, but could be included as a course.